chess/src/test/java/pieces/BoardTest.java

```diff
@@ -3,27 +3,26 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import chess.Board;
-import chess.Color;
 import chess.Piece;
-import chess.Type;
+
 import org.junit.Before;
 import org.junit.Test;
-import static utils.StringUtils.appendNewLine;
+import org.junit.jupiter.api.DisplayName;
 
-import static org.junit.Assert.*;
+import static utils.StringUtils.appendNewLine;
 
 public class BoardTest {
     private Board board;
 
     @Before
     public void setup() {
         board = new Board();
+        board.initializePawns();
+        board.initializeBoard();
```

Comment:
initBoard() 하나로 초기화가 완료되는게 좋지 않을까요? 
```suggestion
        board.initializeBoard();
```